clients/spark/core/src/test/scala/io/treeverse/gc/UncommittedGarbageCollectorSpec.scala

```diff
@@ -148,25 +148,63 @@ class UncommittedGarbageCollectorSpec
           UncommittedGarbageCollector.getFirstSlice(dataDF, repo) should be("")
         })
       }
+
+      it("should return empty first slice with only staged objects") {
+        withSparkSession(_ => {
+          val dataDir = new File(dir.toFile, "data")
+          dataDir.mkdir()
+          val legacySlice = repo + "_legacy_physical:address_path"
```

Comment:
suggest to change the name - legacyPath or something similar

Reply:
done